modules/sql-engine/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -650,3 +650,37 @@ SqlTypeNameSpec IgniteDateTimeTypeName() :
         return new SqlBasicTypeNameSpec(typeName, precision, s.end(this));
     }
 }
+
+SqlNode SqlStartTransaction() :
+{
+    final Span s;
+    IgniteSqlStartTransactionMode mode = IgniteSqlStartTransactionMode.IMPLICIT_READ_WRITE;
+}
+{
+    <START> { s = span(); }
+    <TRANSACTION>
+    [
+        <READ>
+        (
+          <ONLY> {
+            mode = IgniteSqlStartTransactionMode.READ_ONLY;
+          }
+          |
+          <WRITE> {
+            mode = IgniteSqlStartTransactionMode.READ_WRITE;
+          }
+        )
```

Comment:
```suggestion
        <READ> <ONLY> {
            mode = IgniteSqlStartTransactionMode.READ_ONLY;
          }
          |
          <READ> <WRITE> {
            mode = IgniteSqlStartTransactionMode.READ_WRITE;
          }
```
Is it more readable now? 
I see `ColumnNameWithSortDirection` uses the same notation for "nulls first"

Reply:
Thanks. Fixed.